docker-compose.client.yml

```diff
@@ -4,9 +4,9 @@ services:
   client:
     build: ./client
     volumes:
-      - "${WORKSPACE:-./workspace}:/usr/src/app/workspace:ro"
+      - "${WORKSPACE:-./workspace}:/usr/src/app/workspace:Z"
       - "${REPORTS:-./workspace/reports}:/usr/src/app/reports:rw"
```

Comment:
Why does this bind mount not require the change?

Reply:
Does making them all `rw` instead of `ro` also solve the issue?

Reply:
What's `Z` mean here 👀

Reply:
@floion ping ^